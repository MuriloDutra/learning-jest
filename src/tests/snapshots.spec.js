const user = {
  name: "Murilo Alves",
  age: 22,
  job: "Programmer",
};

test("user matches", () => {
  expect(user).toMatchSnapshot();
});
