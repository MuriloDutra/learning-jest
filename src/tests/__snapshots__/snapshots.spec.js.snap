// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`user matches 1`] = `
{
  "age": 22,
  "job": "Programmer",
  "name": "Murilo Alves",
}
`;
